Copy to SD Card root directory to update/config.ini

```diff
@@ -227,6 +227,12 @@ files_sort_by:0
 #   Options: [disable: 0, enable: 1]
 files_list_mode:1
 
+#### Files Fullname View
+# Display fullname for files in List Mode / Icon Mode menu.
+# If disabled, any file extension starting with ".g" or ".G" (e.g. ".g", ".gco", ".gcode" etc.) will be hidden.
+#   Options: [disable: 0, enable: 1]
+files_fullname_view:1
```

Comment:
change the name to `show_filename_extensions`. People are more familiar with this name.

Reply:
changed to "filename_extension"